Tasks/JenkinsQueueJobV2/task.json

```diff
@@ -110,6 +110,24 @@
             "required": false,
             "helpMarkDown": "Specifies strictness of a success definition: whether to consider unstable as a failure or not. False for non-strict, and true for strict version.",
             "groupName": "advanced"
+        },
+        {
+            "name": "retryCount",
+            "type": "string",
+            "label": "Number of retries for failed connection",
+            "defaultValue": "3",
+            "required": false,
+            "helpMarkDown": "Specify number of retries on errors or failures. Default is 3 times",
+            "groupName": "advanced"
+        },
+        {
+            "name": "delayBetweenRetries",
+            "type": "string",
+            "label": "Time between retries",
+            "defaultValue": "60",
```

Comment:
Please align measure unit also with already existing input for other tasks.
Could it make sense to align default value also?

Reply:
Not sure if thats a good idea, since we might have a pretty big timeouts set by users, so the values like 180000 would look strange
I think leaving it as seconds might be a little bit more convenient